shell/platform/windows/platform_handler.cc

```diff
@@ -0,0 +1,121 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/shell/platform/windows/platform_handler.h"
+
+#include <windows.h>
+#include <iostream>
+
+#include "flutter/shell/platform/common/cpp/client_wrapper/include/flutter/json_method_codec.h"
+
+static constexpr char kChannelName[] = "flutter/platform";
+
+static constexpr char kGetClipboardDataMethod[] = "Clipboard.getData";
+static constexpr char kSetClipboardDataMethod[] = "Clipboard.setData";
+
+static constexpr char kTextPlainFormat[] = "text/plain";
+static constexpr char kTextKey[] = "text";
+
+static constexpr char kUnknownClipboardFormatError[] =
+    "Unknown clipboard format error";
+
+namespace flutter {
+
+PlatformHandler::PlatformHandler(flutter::BinaryMessenger* messenger,
+                                 FlutterWindow* window)
+    : channel_(std::make_unique<flutter::MethodChannel<rapidjson::Document>>(
+          messenger,
+          kChannelName,
+          &flutter::JsonMethodCodec::GetInstance())),
+      window_(window) {
+  channel_->SetMethodCallHandler(
+      [this](
+          const flutter::MethodCall<rapidjson::Document>& call,
+          std::unique_ptr<flutter::MethodResult<rapidjson::Document>> result) {
+        HandleMethodCall(call, std::move(result));
+      });
+}
+
+void PlatformHandler::HandleMethodCall(
+    const flutter::MethodCall<rapidjson::Document>& method_call,
+    std::unique_ptr<flutter::MethodResult<rapidjson::Document>> result) {
+  const std::string& method = method_call.method_name();
+  if (method.compare(kGetClipboardDataMethod) == 0) {
+    // Only one string argument is expected.
+    const rapidjson::Value& format = method_call.arguments()[0];
+
+    if (strcmp(format.GetString(), kTextPlainFormat) != 0) {
+      result->Error(kUnknownClipboardFormatError,
+                    "Windows clipboard API only supports text.");
+      return;
+    }
+
+    auto clipboardData = GetClipboardString();
+
+    if (clipboardData.empty()) {
+      result->Error(kUnknownClipboardFormatError,
+                    "Failed to retrieve clipboard data from GLFW api.");
```

Comment:
Copypasta. Maybe search your entire PR for instances of "GLFW"

Reply:
Fixed in next commit